content/en/docs/reference/command-line-tools-reference/feature-gates/index.md

```diff
@@ -0,0 +1,527 @@
+---
+title: Feature Gates
+weight: 10
+content_type: concept
+card:
+  name: reference
+  weight: 60
+---
+
+<!-- overview -->
+This page contains an overview of the various feature gates an administrator
+can specify on different Kubernetes components.
+
+See [feature stages](#feature-stages) for an explanation of the stages for a feature.
+
+
+<!-- body -->
+## Overview
+
+Feature gates are a set of key=value pairs that describe Kubernetes features.
+You can turn these features on or off using the `--feature-gates` command line flag
+on each Kubernetes component.
+
+Each Kubernetes component lets you enable or disable a set of feature gates that
+are relevant to that component.
+Use `-h` flag to see a full set of feature gates for all components.
+To set feature gates for a component, such as kubelet, use the `--feature-gates`
+flag assigned to a list of feature pairs:
+
+```shell
+--feature-gates=...,GracefulNodeShutdown=true
+```
+
+The following tables are a summary of the feature gates that you can set on
+different Kubernetes components.
+
+- The "Since" column contains the Kubernetes release when a feature is introduced
+  or its release stage is changed.
+- The "Until" column, if not empty, contains the last Kubernetes release in which
+  you can still use a feature gate.
+- If a feature is in the Alpha or Beta state, you can find the feature listed
+  in the [Alpha/Beta feature gate table](#feature-gates-for-alpha-or-beta-features).
+- If a feature is stable you can find all stages for that feature listed in the
+  [Graduated/Deprecated feature gate table](#feature-gates-for-graduated-or-deprecated-features).
+- The [Graduated/Deprecated feature gate table](#feature-gates-for-graduated-or-deprecated-features)
+  also lists deprecated and withdrawn features.
+
+{{< note >}}
+For a reference to old feature gates that are removed, please refer to
+[feature gates removed](/docs/reference/command-line-tools-reference/feature-gates-removed/).
+{{< /note >}}
+
+### Feature gates for Alpha or Beta features
+
+{{< table caption="Feature gates for features in Alpha or Beta states" >}}
+
+| Feature | Default | Stage | Since | Until |
+|---------|---------|-------|-------|-------|
+| `APIListChunking` | `false` | Alpha | 1.8 | 1.8 |
```

Comment:
Perhaps in the future, it might be possible to render this table by utilizing another shortcode and including the default value, stage, and release information within the feature gate markdown files.

Reply:
A while ago, I investigated that - see https://github.com/kubernetes/website/pull/28036

This PR is an enabling piece of work for the bigger automation picture.